ts/src/base/Exchange.ts

```diff
@@ -2961,6 +2961,7 @@ export default class Exchange {
             const time = await this.fetchTime (params);
             this.status = this.extend (this.status, {
                 'updated': time,
+                'info': time,
```

Comment:
if we add this we should remove this if below:
```
        if (!('info' in this.status)) {
            this.status['info'] = undefined;
        }
``

Reply:
I dont think we should remove it. if exchange does not have fetch time, we return puppet `this.status` which is created in constructor, but without `info`. so, the structure needs to have `info` in any case

Reply:
i remind in the past we wanted to add `info` in constructor, but then we decided to add it here for some reasons.

Reply:
@ttodua you're right nvm